[Bug] AUR setup fails with compile error

## Description

AUR setup fails with compile error
```
  --> /home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3/target/release/build/whisper-rs-sys-f0fb92fce7ba7ccb/out/bindings.rs:375:26
    |
375 |     ["Size of _IO_FILE"][::std::mem::size_of::<_IO_FILE>() - 216usize];
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of `_` failed here

error[E0080]: attempt to compute `1_usize - 296_usize`, which would overflow
    --> /home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3/target/release/build/whisper-rs-sys-f0fb92fce7ba7ccb/out/bindings.rs:4914:37
     |
4914 |     ["Size of whisper_full_params"][::std::mem::size_of::<whisper_full_params>() - 296usize];
     |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of `_` failed here

```

## Steps to Reproduce

1. paru -S voxtype
2. See errors

## Expected Behavior

Installer to work

## Actual Behavior

Error in compilation

## Environment

- **OS/Distro**:  CachyOS
- **Compositor**: KDE Wayland
- **Voxtype version**: 0.6.3-1
- **Installation method**: AUR
- **Whisper model**: N/A

## Logs

Run `voxtype -vv` and paste relevant output:

```
13:40:13  ~ ❯ paru -S voxtype
:: Resolving dependencies...
:: There are 2 providers available for voxtype:
:: Repository AUR:
    1) voxtype  2) voxtype-bin  
Enter a number (default=1): 1
:: Calculating conflicts...
:: Calculating inner conflicts...

Aur (1)      Old Version  New Version    Make Only
aur/voxtype               0.6.3-1        No

:: Proceed to review? [Y/n]: y

:: Downloading PKGBUILDs...
 PKGBUILDs up to date
 nothing new to review
fetching devel info...
==> Making package: voxtype 0.6.3-1 (Sat 14 Mar 2026 01:40:24 PM CDT)
==> Retrieving sources...
  -> Found voxtype-0.6.3.tar.gz
==> Validating source files with sha256sums...
    voxtype-0.6.3.tar.gz ... Skipped
==> Making package: voxtype 0.6.3-1 (Sat 14 Mar 2026 01:40:25 PM CDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found voxtype-0.6.3.tar.gz
==> Validating source files with sha256sums...
    voxtype-0.6.3.tar.gz ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting voxtype-0.6.3.tar.gz with bsdtar
==> Starting prepare()...
==> Sources are ready.
voxtype-0.6.3-1: parsing pkg list...
==> Making package: voxtype 0.6.3-1 (Sat 14 Mar 2026 01:40:28 PM CDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Removing existing $pkgdir/ directory...
==> Starting build()...
=== BUILD ENVIRONMENT DEBUG ===
cargo: /usr/bin/cargo
cargo 1.94.0 (85eff7c80 2026-01-15)
rustc: /usr/bin/rustc
rustc 1.94.0 (4a4ef493e 2026-03-02)
clang: /usr/bin/clang
clang version 22.1.1
cmake: /usr/bin/cmake
cmake version 4.2.3
PATH=/home/mikhail/.local/share/mise/installs/npm/11.11.1/bin:/home/mikhail/.local/share/mise/installs/go/1.26.1/bin:/home/mikhail/.local/share/mise/installs/node/24.14.0/bin:/home/mikhail/.local/share/mise/installs/python/3.14.3/bin:/home/mikhail/.local/share/mise/installs/uv/0.10.10/uv-x86_64-unknown-linux-musl:/nix/var/nix/profiles/default/bin:/home/mikhail/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
RUSTUP_TOOLCHAIN=unset
CARGO_HOME=/home/mikhail/.cargo
RUSTUP_HOME=/home/mikhail/.rustup
CFLAGS=-march=native -O3 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -flto=auto
CXXFLAGS=-march=native -O3 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -flto=auto
LDFLAGS=-Wl,-O1 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now          -Wl,-z,pack-relative-relocs -flto=auto
DEBUG_RUSTFLAGS=-C debuginfo=2
RUSTFLAGS=-C opt-level=3 -C target-cpu=native
RUSTUP_HOME=/home/mikhail/.rustup
===============================
   Compiling proc-macro2 v1.0.103
   Compiling unicode-ident v1.0.22
   Compiling quote v1.0.42
   Compiling libc v0.2.177
   Compiling cfg-if v1.0.4
   Compiling shlex v1.3.0
   Compiling find-msvc-tools v0.1.5
   Compiling stable_deref_trait v1.2.1
   Compiling once_cell v1.21.3
   Compiling smallvec v1.15.1
   Compiling serde_core v1.0.228
   Compiling glob v0.3.3
   Compiling pkg-config v0.3.32
   Compiling autocfg v1.5.0
   Compiling prettyplease v0.2.37
   Compiling regex-syntax v0.8.8
   Compiling minimal-lexical v0.2.1
   Compiling litemap v0.8.1
   Compiling cc v1.2.47
   Compiling bitflags v2.10.0
   Compiling writeable v0.6.2
   Compiling memchr v2.7.6
   Compiling serde v1.0.228
   Compiling libloading v0.8.9
   Compiling bindgen v0.71.1
   Compiling utf8parse v0.2.2
   Compiling icu_normalizer_data v2.1.1
   Compiling clang-sys v1.8.1
   Compiling either v1.15.0
   Compiling log v0.4.28
   Compiling icu_properties_data v2.1.2
   Compiling anstyle-parse v0.2.7
   Compiling zerocopy v0.8.33
   Compiling heck v0.5.0
   Compiling anstyle-query v1.1.5
   Compiling itertools v0.13.0
   Compiling zeroize v1.8.2
   Compiling is_terminal_polyfill v1.70.2
   Compiling nom v7.1.3
   Compiling colorchoice v1.0.4
   Compiling rustc-hash v2.1.1
   Compiling anstyle v1.0.13
   Compiling version_check v0.9.5
   Compiling rustls-pki-types v1.13.2
   Compiling alsa-sys v0.3.1
   Compiling syn v2.0.111
   Compiling anstream v0.6.21
   Compiling parking_lot_core v0.9.12
   Compiling strsim v0.11.1
   Compiling clap_lex v0.7.6
   Compiling fs_extra v1.3.0
   Compiling ahash v0.8.12
   Compiling bitflags v1.3.2
   Compiling clap_builder v4.5.53
   Compiling aho-corasick v1.1.4
   Compiling memoffset v0.6.5
   Compiling cmake v0.1.54
   Compiling untrusted v0.9.0
   Compiling rustix v1.1.2
   Compiling scopeguard v1.2.0
   Compiling regex-automata v0.4.13
   Compiling crc32fast v1.5.0
   Compiling thiserror v1.0.69
   Compiling radium v0.7.0
   Compiling zmij v1.0.12
   Compiling crossbeam-utils v0.8.21
   Compiling ring v0.17.14
   Compiling vcpkg v0.2.15
   Compiling getrandom v0.2.16
   Compiling pin-project-lite v0.2.16
   Compiling inotify-sys v0.1.5
   Compiling lock_api v0.4.14
   Compiling num-traits v0.2.19
   Compiling tracing-core v0.1.35
   Compiling cexpr v0.6.0
   Compiling simd-adler32 v0.3.8
   Compiling adler2 v2.0.1
   Compiling cpal v0.15.3
   Compiling utf8_iter v1.0.4
   Compiling serde_json v1.0.149
   Compiling libsqlite3-sys v0.30.1
   Compiling getrandom v0.3.4
   Compiling hashbrown v0.16.1
   Compiling cfg_aliases v0.2.1
   Compiling tap v1.0.1
   Compiling equivalent v1.0.2
   Compiling linux-raw-sys v0.11.0
   Compiling rustls v0.23.36
   Compiling roff v0.2.2
   Compiling percent-encoding v2.3.2
   Compiling regex v1.12.2
   Compiling getrandom v0.4.1
   Compiling form_urlencoded v1.2.2
   Compiling hashbrown v0.14.5
   Compiling wyz v0.5.1
   Compiling nix v0.29.0
   Compiling miniz_oxide v0.8.9
   Compiling parking_lot v0.12.5
   Compiling indexmap v2.12.1
   Compiling alsa v0.9.1
   Compiling signal-hook-registry v1.4.7
   Compiling mio v1.1.0
   Compiling socket2 v0.6.1
   Compiling webpki-roots v1.0.5
   Compiling funty v2.0.0
   Compiling subtle v2.6.1
   Compiling bytes v1.11.0
   Compiling winnow v0.7.14
   Compiling itoa v1.0.17
   Compiling synstructure v0.13.2
   Compiling dasp_sample v0.11.0
   Compiling lazy_static v1.5.0
   Compiling whisper-rs v0.15.1
   Compiling option-ext v0.2.0
   Compiling same-file v1.0.6
   Compiling toml_write v0.1.2
   Compiling anyhow v1.0.100
   Compiling walkdir v2.5.0
   Compiling dirs-sys v0.4.1
   Compiling sharded-slab v0.1.7
   Compiling bitvec v1.0.1
   Compiling zerofrom-derive v0.1.6
   Compiling yoke-derive v0.8.1
   Compiling zerovec-derive v0.11.2
   Compiling displaydoc v0.2.5
   Compiling serde_derive v1.0.228
   Compiling clap_derive v4.5.49
   Compiling thiserror-impl v1.0.69
   Compiling tracing-attributes v0.1.31
   Compiling tokio-macros v2.6.0
   Compiling zerofrom v0.1.6
   Compiling webpki-roots v0.26.11
   Compiling yoke v0.8.1
   Compiling crossbeam-channel v0.5.15
   Compiling zerovec v0.11.5
   Compiling zerotrie v0.2.3
   Compiling matchers v0.2.0
   Compiling tokio v1.48.0
   Compiling flate2 v1.1.5
   Compiling tinystr v0.8.2
   Compiling potential_utf v0.1.4
   Compiling tracing v0.1.41
   Compiling icu_locale_core v2.1.1
   Compiling icu_collections v2.1.1
   Compiling hashlink v0.9.1
   Compiling clap v4.5.53
   Compiling clap_mangen v0.2.31
   Compiling nix v0.23.2
   Compiling tracing-log v0.2.0
   Compiling inotify v0.9.6
   Compiling voxtype v0.6.3 (/home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3)
   Compiling mio v0.8.11
   Compiling filetime v0.2.26
   Compiling icu_provider v2.1.1
   Compiling thread_local v1.1.9
   Compiling hound v3.5.1
   Compiling fastrand v2.3.0
   Compiling env_home v0.1.0
   Compiling icu_properties v2.1.2
   Compiling icu_normalizer v2.1.1
   Compiling nu-ansi-term v0.50.3
   Compiling iana-time-zone v0.1.65
   Compiling futures-core v0.3.31
   Compiling base64 v0.22.1
   Compiling fallible-streaming-iterator v0.1.9
   Compiling fallible-iterator v0.3.0
   Compiling serde_spanned v0.6.9
   Compiling toml_datetime v0.6.11
   Compiling chrono v0.4.43
   Compiling tracing-subscriber v0.3.20
warning: voxtype@0.6.3: Man pages generated in: /home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3/target/release/build/voxtype-397e8a28ba09e141/out/man
   Compiling tempfile v3.23.0
   Compiling rodio v0.19.0
   Compiling idna_adapter v1.2.1
   Compiling idna v1.1.0
   Compiling toml_edit v0.22.27
   Compiling which v7.0.3
   Compiling url v2.5.8
   Compiling notify v6.1.1
   Compiling uuid v1.21.0
   Compiling whisper-rs-sys v0.14.1
   Compiling evdev v0.12.2
   Compiling rustls-webpki v0.103.8
   Compiling toml v0.8.23
   Compiling inotify v0.10.2
   Compiling async-trait v0.1.89
   Compiling directories v5.0.1
   Compiling pidlock v0.1.6
   Compiling num_cpus v1.17.0
   Compiling ureq v2.12.1
error[E0080]: attempt to compute `1_usize - 216_usize`, which would overflow
   --> /home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3/target/release/build/whisper-rs-sys-f0fb92fce7ba7ccb/out/bindings.rs:375:26
    |
375 |     ["Size of _IO_FILE"][::std::mem::size_of::<_IO_FILE>() - 216usize];
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of `_` failed here

error[E0080]: attempt to compute `1_usize - 296_usize`, which would overflow
    --> /home/mikhail/.cache/paru/clone/voxtype/src/voxtype-0.6.3/target/release/build/whisper-rs-sys-f0fb92fce7ba7ccb/out/bindings.rs:4914:37
     |
4914 |     ["Size of whisper_full_params"][::std::mem::size_of::<whisper_full_params>() - 296usize];
     |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of `_` failed here

For more information about this error, try `rustc --explain E0080`.
error: could not compile `whisper-rs-sys` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'voxtype-0.6.3-1': 
error: packages failed to build: voxtype-0.6.3-1
```

## Additional Context

Any other information that might help (screenshots, config file snippets, etc.)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -64,7 +64,7 @@ tempfile = "3"
 rodio = { version = "0.19", default-features = false, features = ["wav"] }
 
 # Whisper speech-to-text
-whisper-rs = "0.15.1"
+whisper-rs = "0.16.0"
 
 # Parakeet speech-to-text (optional, ONNX-based)
 parakeet-rs = { version = "0.3", optional = true }

